apypie/api.py

```diff
@@ -132,18 +132,18 @@ def call(self, resource_name, action_name, params={}, headers={}, options={}):
         if not options.get('skip_validation', False):
             action.validate(params)
 
-        return self.call_action(action, params, headers, options)
+        return self._call_action(action, params, headers, options)
 
-    def call_action(self, action, params={}, headers={}, options={}):
+    def _call_action(self, action, params={}, headers={}, options={}):
         route = action.find_route(params)
         get_params = dict((key, value) for key, value in params.items() if key not in route.params_in_path)
-        return self.http_call(
+        return self._http_call(
             route.method,
             route.path_with_params(params),
             get_params,
```

Comment:
I'd like to keep that one public for cases where we might want to make a direct http call, e.g. when calling an action that is not properly documented.

See https://github.com/Apipie/apipie-bindings/blob/master/lib/apipie_bindings/api.rb#L197

Reply:
Stupid FastHub client.. the comment is about http_call()